bin/pepProxy

```diff
@@ -64,7 +65,7 @@ function loadConfiguration() {
 
     for (var i = 0; i < environmentValues.length; i++) {
         if (process.env[environmentValues[i]]) {
-            console.log('Changing %s to environment value: %s', environmentValues[i], process.env[environmentValues[i]]);
```

Comment:
Do we have console.log yet in the code? :O

Taking advantage of this PR... could you do a overall search for them to check if there are some other in the code, please?

Reply:
Those are the only 4 references found in the code. I assume it was due to `bin/pepProxy` is a bit special file.

Reply:
Maybe it should be noted in CNR, eg:

```
- Fix: use logops library instead of direct console.log printing in all cases
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-pep-steelskin/pull/487/commits/80faac1e9ac56acb8ef385f8032de6f10472ded5